src/styles/blocks/header.scss

```diff
@@ -0,0 +1,57 @@
+.header {
+  height: 60px;
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.nav {
+  height: 60px;
+  display: flex;
```

Comment:
You have separate `nav` block - therefore its styles should be in a separate file.